flink-runtime-web/web-dashboard/src/app/pages/job/exceptions/job-exceptions.component.html

```diff
@@ -53,6 +54,9 @@
               <td>
                 <div class="name">{{ item.selected.exceptionName }}</div>
               </td>
+              <td>
+                {{ item.selected.failureLabels ? (item.selected.failureLabels | json) : 'None' }}
```

Comment:
🤔 Have you looked into the tag component of NG-zorro (the underlying UI library we're using)? That could look quite nice.

https://ng.ant.design/components/tag/en

Reply:
@dmvk Do you mean to add a tag component to json here?